README.md

```diff
@@ -1,16 +1,81 @@
-# AceBook
+# Acebook
 
-Fork this repository to `acebook-teamname` and customize
-the below**
+## Summary
 
-[You can find the engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
+Week 8 and 9 Makers Academy engineering project to develop an Facebook clone, called Acebook. <br>
+You can find the engineering project outline [here](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
+___
 
-The card wall is here: <please update>
+## Credits
 
-## How to contribute to this project
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+Developed in 10 days by Daniel Lau, Elishka Flint, Jeff Hung and Jay Khan.
+Reflections on process and learnings from our first week of group-work can be found [here](https://hackmd.io/-5Q6g-SnQqaELTfrZjOP0w?both).
+___
 
-## Quickstart
+## Objectives
+
+Agreed as a team in our first planning meeting:
+* Tbc
+___
+
+## State of Completion
+
+### Implemented Functionality
+
+#### Front-end
+* tbc
+
+#### Back-end
+* tbc
+
+### What we would do next
+* tbc
+___
+
+## User Stories:
+
+### First Sprint (MVP)
+
+```
+As a user,
+So that I can use Acebook,
+I want to be able to sign up for an account
+```
+```
+As a user,
+So that I can post on Acebook,
+I want to be able to add some text to my homepage
+```
+```
+As a user,
+So that I can see what's going on,
+I want to be able to see other people's posts on my homepage
+```
+### Next user stories
+```
+As a user,
+So that I can come back to Acebook again and again,
+I want to be able to sign in and out of sessions
+```
+```
+As a user,
+So that I can keep track of my activity,
+I want to be able to view my own profile with my posts only
+```
+```
+As a user,
+So that I can curate my content,
+I want to be able to update and delete my posts
+```
+___
+
+## Technical
+
+### Tech/Frameworks Used
+
+Ruby on Rails, Sinatra, CSS, RSpec, Capybara
```

Comment:
I don't think Sinatra is part of the Tech Stack, the server running seems to be Puma (when you run 'bin/rails server' in the command line). Also, there's no Sinatara gem in the Gemfile

Reply:
Thanks - updated